java/app/src/main/AndroidManifest.xml

```diff
@@ -2,13 +2,17 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.fullstorydev.shoppedemo">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme" >
+        android:theme="@style/AppTheme"
+        android:networkSecurityConfig="@xml/network_security_config">
```

Comment:
since `network_security_config` is .gitignored, will adding it to the manifest here cause breakage if it's not present in the project (think: someone forgot to add it when they cloned the repo)

Reply:
should we not ignore the file (expose the URL in this repo)?
Also applies to the hosts files that I had added to gitignore for links that I did not want to be exposed to github(but maybe we should actually expose them)?
http://reactshoppe.s3-website-us-east-1.amazonaws.com/data/products.json
https://fruitshoppe.firebaseapp.com/images/

Reply:
I think you're ok. I see now that you've added the `network_security_config` and there's plenty of warnings about not opening network activity to non-SSL endpoints in your comments. Go ahead and make an issue on your repo to expose an SSL endpoint for the product catalog so you can remove this config in the future.